step-release-vis/src/app/components/environment/environment_test.ts

```diff
@@ -72,6 +72,27 @@ describe('EnvironmentComponent', () => {
         expect(component.polygons[i].highlight).toBeFalse();
       }
     });
+
+    it('tooltip should show, move position and disappear', () => {
```

Comment:
fine for now, but usually having a sinlge assert section per test is better practice

Reply:
sorry, I have to argue with that. I would like to see this as 3 separate testcases. Setup part can go to a describe's foreach section, so not much of a duplication, but much more cleaner and readable code.